docs/general/options.md

```diff
@@ -0,0 +1,48 @@
+# Options
+
+## Scriptable Options
+
+Scriptable options also accept a function which is called for each data and that takes the unique argument `context` representing contextual information (see [option context](options.md#option-context)).
+
+Example:
+
+```javascript
+color: function(context) {
```

Comment:
To illustrate my previous message: let's say a third party library provides the following function:

```javascript
function colorize(i) {
	return color_palette[i % color_palette.length];
}
```

Then, if `index` is provided, it's easy to plug an option to that function:
```javascript
point: {
    color: colorize
}
```

Instead of:
```javascript
point: {
    color: function(context) {
        return colorize(context.dataIndex);
    }
}
```

However, not sure how this use case is common or will work as-is most of the time, and having this first `index` argument might be burden later if we realize it's more confusing than helpful. So I *think* `function(context)` is better, just want to be sure we are all on the same page.